### ISSUE TITLE
Unable to view the workflow page

### ISSUE THREAD

**Issue by LinzeSu** [kubevela/velaux#686]
**Describe the bug**
As the title suggest.

**To Reproduce**
Create an application. One of the step failed and I want to check the reason in this page. And it won't get ready.

**Screenshots**
![velaux](https://user-images.githubusercontent.com/87755575/223049441-6f69bd30-b58a-43af-b2b5-9d7b7c16f283.png)

**KubeVela Version**
1.7.3

**Additional context**
Used both port-forward and nodeport way. But both did not work.

**Comment by barnettZQG** [comment]
@LinzeSu How do you create the application? I see that the application environment list can't load successfully.

**Comment by LinzeSu** [comment]
I created the app via velaux API.

**Comment by barnettZQG** [comment]
> I created the app via velaux API.

Your page may not be loaded completely when you screenshot. You could check all API requests.

**Comment by LinzeSu** [comment]
> Your page may not be loaded completely when you screenshot. 

The same page for addon-fluxcd loads pretty fast. So I wonder there might be problems in specific application.

> You could check all API requests.

Are you suggesting getting components status using API?

**Comment by barnettZQG** [comment]
> Are you suggesting getting components status using API?
No. The page keeps loading status, you could check the API requests via the browser console. focus follow APIs:

1. `/api/v1/applications/test-app1/envs`
2. `/api/v1/applications/test-app1/workflows`

**Comment by LinzeSu** [comment]
>focus follow APIs:
>/api/v1/applications/test-app1/envs
>/api/v1/applications/test-app1/workflows

Both of the API returned:
```
<Bcode>
  <HTTPCode>401</HTTPCode>
  <BusinessCode>12006</BusinessCode>
  <Message>the user is not authorized</Message>
</Bcode>
```
I logged in as the admin user. And there's no other users. What could this suggest?

**Comment by barnettZQG** [comment]
I think this error is from your custom client code. This is irrelevant to this issue. So, can we have insight into the UI error first?  Watch the API request via the browser.

**Comment by LinzeSu** [comment]
Sorry that the text font is small. The /api/v1/applications/test-app1/envs worked fine and the /api/v1/applications/test-app1/workflows got error cause the message "the user is not authorized".
![velaux](https://user-images.githubusercontent.com/87755575/223074005-22c5c3dc-e535-4a7d-80f2-1ae76d6d8661.png)

**Comment by barnettZQG** [comment]
@LinzeSu Can you print the results of the above APIs?

**Comment by LinzeSu** [comment]
@barnettZQG Sure. The response body of workflows is:
```
{
 "workflows": [
  {
   "name": "test-workflow",
   "alias": "",
   "description": "",
   "enable": false,
   "default": false,
   "envName": "default",
   "createTime": "2023-03-06T10:59:10.055164314Z",
   "updateTime": "2023-03-06T10:59:10.055164635Z",
   "mode": "StepByStep",
   "subMode": "DAG",
   "steps": [
    {
     "name": "install-elastic-operator",
     "alias": "",
     "type": "apply-component",
     "description": "",
     "dependsOn": [],
     "properties": {
      "component": "elastic-operator",
      "policies": [
       "test-policy1"
      ]
     }
    },
    {
     "name": "install-strimzi-operator",
     "alias": "",
     "type": "apply-component",
     "description": "",
     "dependsOn": [],
     "properties": {
      "component": "strimzi-operator",
      "policies": [
       "test-policy1"
      ]
     }
    },
    {
     "name": "install-spark-operator",
     "alias": "",
     "type": "apply-component",
     "description": "",
     "dependsOn": [],
     "properties": {
      "component": "spark-operator",
      "policies": [
       "test-policy1"
      ]
     }
    }
   ]
  },
  {
   "name": "workflow-default",
   "alias": "Default Workflow",
   "description": "Created automatically by envbinding.",
   "enable": false,
   "default": true,
   "envName": "default",
   "createTime": "2023-03-06T10:59:08.977875821Z",
   "updateTime": "2023-03-06T10:59:09.618425637Z",
   "mode": "StepByStep",
   "subMode": "DAG",
   "steps": [
    {
     "name": "default",
     "alias": "Deploy To default",
     "type": "deploy",
     "description": "deploy app to delivery target default",
     "dependsOn": null,
     "properties": {
      "policies": [
       "default"
      ]
     }
    }
   ]
  }
 ]
}
```
the envs:
```
{
 "envBindings": null
}
```

**Comment by barnettZQG** [comment]
The reason is found. This application doesn't bind any environments. Let's try:

1. Delete all workflows on the workflow page.
2. Delete all policies on the basic info page.
3. Bind the default environment.

**Comment by LinzeSu** [comment]
Thanks. Will try these steps. But two quick questions before actually going:
1. I speficied envBinding field to "default" in the API according to API documentation https://kubevela.stoplight.io/docs/kubevela/1b209e9ff332e-create-one-application. Why is it not bind to deafult env?
2. Is there a way to specify a application namespace? I didn't see this field in the API documentation,
